src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,35 @@
+.calendar {
+  margin: 0 auto;
+  width: $cell-width * 7 + (6 * $gap);
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: flex-end;
+  gap: $gap;
+  padding-top: 50px;
+
+  &__cell {
+    position: relative;
+    width: 98px;
+    height: 98px;
```

Comment:
You can create a variable for the cell size as well. Or think about the mixin here